src/net/http/cookie.go

```diff
@@ -267,6 +274,11 @@ func (c *Cookie) Valid() error {
 			return errors.New("http: invalid Cookie.Domain")
 		}
 	}
+	if c.Partitioned {
+		if !c.Secure || c.Path != "/" {
+			return errors.New("http: partitioned cookies must be set with Secure and Path=/")
```

Comment:
From my understanding, partitioned cookie's path doesn't have to be `/`. Only __Host prefixed cookie must be both secure and path=/.

> Partitioned cookies must be set with Secure.
It is recommended to use the __Host prefix when setting partitioned cookies to make them bound to the hostname (and not the registrable domain).
https://developers.google.com/privacy-sandbox/3pcd/chips